packages/autocomplete/src/elements/Autocomplete.spec.js

```diff
@@ -6,118 +6,104 @@
  */
 
 import React from 'react';
-import { Label, FauxInput, Input } from '@zendeskgarden/react-textfields';
-import { MenuView, Item } from '@zendeskgarden/react-menus';
-import { mountWithTheme } from '@zendeskgarden/react-testing';
+import { render, fireEvent } from 'garden-test-utils';
 import { KEY_CODES } from '@zendeskgarden/react-selection';
 import Autocomplete from './Autocomplete';
 
 describe('Autocomplete', () => {
   it('renders children if provided', () => {
-    const wrapper = mountWithTheme(
+    const { getByTestId } = render(
       <Autocomplete options={[]}>
-        <div id="autocomplete-child">Child content</div>
+        <div data-test-id="autocomplete-child">Child content</div>
       </Autocomplete>
     );
 
-    expect(wrapper.find('#autocomplete-child')).toHaveText('Child content');
+    expect(getByTestId('autocomplete-child').textContent).toBe('Child content');
   });
 
   it('renders label if provided', () => {
     const label = 'Test Label';
-    const wrapper = mountWithTheme(<Autocomplete label={label} options={[]} />);
+    const { getByTestId } = render(
+      <Autocomplete label={<span data-test-id="label">{label}</span>} options={[]} />
+    );
 
-    expect(wrapper.find(Label)).toHaveText(label);
+    expect(getByTestId('label').textContent).toBe(label);
   });
 
   it('renders aria-label if provided', () => {
     const ariaLabel = 'Test Aria Label';
-    const wrapper = mountWithTheme(<Autocomplete aria-label={ariaLabel} options={[]} />);
+    const { container } = render(<Autocomplete aria-label={ariaLabel} options={[]} />);
 
-    expect(wrapper.find(FauxInput)).toHaveProp('aria-label', ariaLabel);
+    expect(container.firstChild.firstChild).toHaveAttribute('aria-label', ariaLabel);
```

Comment:
since it's just DOM you could do `container.querySelector('label')`

Reply:
This is actually testing for an `aria-label` value so there is no semantic `<label>` to query for. This is the same way we get to the `<FauxInput>` in other tests.